tests/test_trame.py

```diff
@@ -412,6 +412,7 @@ async def stop(*args, **kwargs):
     """)
 
 
+@importorskip("vtk")
```

Comment:
`vtk` is dist name, importable package is `vtkmodules` (as in the error message of the failed test).